src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,70 @@ Special packages
 ================
 
 BLAS
```

Comment:
If you decide to move the paragraphs on switching the blas implementation to the user guide  (see comment below), I think the remaining would fit better into the "special dependencies" section.